client/js/components/App.react.jsx

```diff
@@ -16,24 +18,51 @@ export default class App extends React.Component {
     constructor(props) {
         super(props);
         this.state = {park: null,
-                      parkList: []};
-        getParksJson().then((data) => this.setState({parkList: data}));
+                      parkList: [],
+                      parkByID: {}};
+        getParksJson().then((data) => {
+            var parkByID = {}
+            data.forEach((current, index) => {
+                // NOTE: Coordinates are lng lat here
```

Comment:
Well coordinates isn't geojson. I grab it from geojson tho. https://github.com/open-austin/austingreenmap/blob/20a7139ae006810f6113304a4c40eba9ad229ff0/generate_parks.py#L114